bot/cogs/trivia.py

```diff
@@ -0,0 +1,307 @@
+                        Cartoons or Cartoon
+                            """,
+                color=EMBED_COLOR
+            )
+            embed.set_footer(
+                text="Do .trivia (category name) to choose a category, categories are NOT case sensitive."
+            )
+            await ctx.send(embed=embed)
+            return
+        elif category == 'general':
+            category = 9
+        elif category == 'book' or category == 'books':
+            category = 10
+        elif category == 'film':
+            category = 11
+        elif category == 'music':
+            category = 12
+        elif category == 'theatre':
+            category = 13
+        elif category == 'television' or category == 'tv':
+            category = 14
+        elif category == 'vg' or category == 'video games':
+            category = 15
+        elif category == 'bg' or category == 'board games':
+            category = 16
+        elif category == 'science':
+            category = 17
+        elif category == 'computer' or category == 'computers':
+            category = 18
+        elif category == 'math':
+            category = 19
+        elif category == 'mythology' or category == 'myth':
+            category = 20
+        elif category == 'sports':
+            category = 21
+        elif category == 'geography' or category == 'geo':
+            category = 22
+        elif category == 'history':
+            category = 23
+        elif category == 'art':
+            category = 25
+        elif category == 'animals' or category == 'animal':
+            category = 27
+        elif category == 'comics' or category == 'comic':
+            category = 29
+        elif category == 'anime':
+            category = 31
+        elif category == 'cartoons' or category == 'cartoon':
+            category = 32
+
+        message = await ctx.send(embed=Embed(title='Loading.....', color=EMBED_COLOR))
+
+        while True:
+            t_data = await self.get_trivia(category)
+            embed = Embed(
+                title=t_data['question'],
+                description=f"1️⃣: {t_data['answers'][0]}\n\n"
+                            f"2️⃣: {t_data['answers'][1]}\n\n"
+                            f"3️⃣: {t_data['answers'][2]}\n\n"
+                            f"4️⃣: {t_data['answers'][3]}\n\n"
+                            f"You have 15 seconds or until {ctx.author.mention} answers!",
+                color=EMBED_COLOR
+            )
+            embed.set_author(name=f"{t_data['difficulty']} question for {t_data['points']} points")
+            embed.set_footer(text=t_data['category'])
+
+            await message.edit(
+                embed=embed
+            )
+            await message.add_reaction('1️⃣')
+            await message.add_reaction('2️⃣')
+            await message.add_reaction('3️⃣')
+            await message.add_reaction('4️⃣')
+
+            answers = {}
+
+            def r_check(reaction: Reaction, user: User) -> bool:
```

Comment:
these arguments shadow outer scope, you should rename them to avoid this. Perhaps something like `check_reaction` and `check_user`